acceptance/helper.rb

```diff
@@ -254,14 +254,14 @@ def display_last_logs(host)
   # @param host Hostname to test for PuppetDB availability
   # @return [void]
   # @api public
-  def sleep_until_started(host)
+  def sleep_until_started(host, test_url="/pdb/query/v4/version", ssl_test_url="/pdb/")
```

Comment:
Might be some curl_with_retries weirdness that comes about, but yeah - this feels like it should be the same thing.